src/backend/joanie/settings.py

```diff
@@ -466,7 +466,8 @@ class Base(Configuration):
         environ_prefix=None,
     )
     JOANIE_SIGNATURE_VALIDITY_PERIOD = values.PositiveIntegerValue(
-        60 * 60 * 24 * 15,
+        99 * 60 * 60 * 24,  # Duration in seconds
```

Comment:
Why not directly use ms here ?

Reply:
Because we are using this settings somewhere else in the app and in those case we are using seconds.

Reply:
Yup agreed, let's explicit that.
I'll just assume that in Python, it's looks common to express duration in seconds.